packages/screens/LaunchpadERC20/hooks/useLastAirdrops.ts

```diff
@@ -9,10 +9,10 @@ import {
 } from "../../../networks";
 
 import { extractGnoJSONString } from "@/utils/gno";
-import { zodAidrop } from "@/utils/launchpadERC20/types";
+import { zodAidrop } from "@/utils/types/types";
 
 export const useLastAirdrops = (networkId: string) => {
-  return useQuery(["lastAirdrops"], async () => {
+  return useQuery(["lastAirdrops", networkId], async () => {
```

Comment:
```suggestion
  return useQuery(["erc20LaunchpadLastAirdrops", networkId], async () => {
```

we should use very explicit query keys prefixes to avoid conflicts in the future as these kind of conflicts are hard to pinpoint